db/migrate/20230104171944_create_bookings.rb

```diff
@@ -0,0 +1,10 @@
+class CreateBookings < ActiveRecord::Migration[7.0]
+  def change
+    create_table :bookings do |t|
+      t.datetime :start_date
+      t.datetime :end_date
```

Comment:
Good job adding this fields, I would suggest if you can rename them to something more conventional like
`pick_up_date` and `return_date` it's more semantic with our app

Reply:
Ow okay, I just used the one in the card

Reply:
No problem, update the card too then 🚀

Reply:
@michael-duke make sure to rename the other fields in the card as well (if u wish so) for the next tasks.